pdf/footer.tpl

```diff
@@ -33,13 +33,13 @@
 
 			{if !empty($shop_phone) OR !empty($shop_fax)}
 				{l s='For more assistance, contact Support:' d='Shop.Pdf' pdf='true'}<br />
-				{if !empty($shop_phone)}
-					{l s='Tel: %s' sprintf=[$shop_phone|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
-				{/if}
+        {if !empty($shop_phone)}
+          {l s='Tel: %s' sprintf=[$shop_phone|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
+        {/if}
 
-				{if !empty($shop_fax)}
-					{l s='Fax: %s' sprintf=[$shop_fax|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
-				{/if}
+        {if !empty($shop_fax)}
+          {l s='Fax: %s' sprintf=[$shop_fax|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
+        {/if}
```

Comment:
Why this change? Indentations look incorrect now

Reply:
Usually when I see such diff in GitHub, it's white characters replaced by another white characters. Like spaces and tabs.

Here I think tabs were replaced by spaces